test/test_transforms.py

```diff
@@ -681,15 +681,224 @@ def test_convert_image_dtype_int_to_int_consistency(self):
                     self.assertEqual(actual_min, desired_min)
                     self.assertEqual(actual_max, desired_max)
 
-    @unittest.skipIf(accimage is None, 'accimage not available')
+
+    def test_affine(self):
+        input_img = np.zeros((40, 40, 3), dtype=np.uint8)
+        cnt = [20, 20]
+        for pt in [(16, 16), (20, 16), (20, 20)]:
+            for i in range(-5, 5):
+                for j in range(-5, 5):
```

Comment:
these 3 for-loops is typically the kind of thing that we will want to parametrize, as done for example in https://github.com/pytorch/vision/pull/3907/files